scripts/insert_into_appointments.sql

```diff
@@ -1 +1,26 @@
 -- insert appointments creating connections between doctors and patients
+insert into appointments (date_of, time_of, patient_id, doctor_id)
+  select timestamp '2016-08-12', 10:00, p.id, d.id
```

Comment:
The time needs to be wrapped in single quotation marks and preceded by `TIME`.  e.g., `TIME '10:00'`.